README.md

```diff
@@ -35,7 +35,23 @@
 
 ## Contributing
 
-We welcome contributions to make this project better. You can submit feature requests or report bugs by creating an [issue](https://github.com/iTwin/saved-views/issues).
+### Issues
```

Comment:
Does it make more sense to link to this repo-level contributing section in each package's README? Rather than copy-pasting it? Maybe it doesn't... Just throwing this out there. I don't know which is more common in other repos (in general, across GitHub).